### PR TITLE
Started deleting .DS_Store files so licenses can run on mac os x.

### PR DESCRIPTION
**Pull request by gaaclarke**

### CODE REVIEW

**Comment by gaaclarke** [comment]
> LGTM.
> 
> As an alternative, is there some way to just tell macOS to _not_ create these files in this tree?

I don't think so.  I looked into changing the script to ignore them but it seemed contrary to the script to ignore files for processing since it wants to check everything for license problems.

File: ci/licenses.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ exitStatus=0
 dart --version
 
 # These files trip up the script on Mac OS X.
-find . -name ".DS_Store" | xargs rm
+find . -name ".DS_Store" -exec rm {} \;
```
Comment (dnfield) [inline ci/licenses.sh]: uber-nit: you can just do `-delete`.

Comment (gaaclarke) [reply ci/licenses.sh]: right on, i would change it but end of day and ci takes forever

Comment (cbracken) [reply ci/licenses.sh]: Don't you need quotes to deal with paths with spaces here?

**Pull request merged**